doc/user/content/self-managed-deployments/installation/install-on-gcp.md

```diff
@@ -0,0 +1,321 @@
+- [kubectl gke plugin](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl#install_plugin)
+
+### License Key
+
+{{< yaml-table data="self_managed/license_key" >}}
+
+## Getting started: Simple example
+
+{{< warning >}}
+
+{{< self-managed/terraform-disclaimer >}}
+
+{{< /warning >}}
+
+### Step 1: Set Up the Environment
+
+1. Open a terminal window.
+
+1. Clone the Materialize Terraform repository and go to the
+   `gcp/examples/simple` directory.
+
+   ```bash
+   git clone https://github.com/MaterializeInc/materialize-terraform-self-managed.git
+   cd materialize-terraform-self-managed/gcp/examples/simple
+   ```
+
+1. Authenticate to GCP with your user account.
+
+   ```bash
+   gcloud auth login
+   ```
+
+1. Find the list of GCP projects:
+
+   ```bash
+   gcloud projects list
+   ```
+
+1. Set your active GCP project, substitute with your `<PROJECT_ID>`.
+
+   ```bash
+   gcloud config set project <PROJECT_ID>
+   ```
+
+1. Enable the following APIs for your project:
+
+   ```bash
+   gcloud services enable container.googleapis.com               # For creating Kubernetes clusters
+   gcloud services enable compute.googleapis.com                 # For creating GKE nodes and other compute resources
+   gcloud services enable sqladmin.googleapis.com                # For creating databases
+   gcloud services enable cloudresourcemanager.googleapis.com    # For managing GCP resources
+   gcloud services enable servicenetworking.googleapis.com       # For private network connections
+   gcloud services enable iamcredentials.googleapis.com          # For security and authentication
+   gcloud services enable iam.googleapis.com                     # For managing IAM service accounts and policies
+   gcloud services enable storage.googleapis.com                 # For Cloud Storage buckets
+   ```
+
+1. Authenticate application default credentials for Terraform
+
+   ```bash
+   gcloud auth application-default login
+   ```
+
+### Step 2: Configure Terraform Variables
+
+1. Create a `terraform.tfvars` file and specify the following variables:
+
+   | Variable      | Description                 |
+   | -----------   | ----------------------------|
+   | `project_id`  | Set to your GCP project ID. |
+   | `name_prefix` | Set a prefix for all resource names (e.g., `simple-demo`) as well as your release name for the Operator |
+   | `region`      | Set the GCP region for the deployment (e.g., `us-central1`).  |
+   | `license_key` | Set to your Materialize license key.     |
+   | `labels`      | Set to the labels to apply to resources. |
+
+   ```bash
+   project_id  = "my-gcp-project"
+   name_prefix = "simple-demo"
+   region      = "us-central1"
+   license_key = "your-materialize-license-key"
+   labels = {
+     environment = "demo"
+     created_by  = "terraform"
+   }
+   # internal_load_balancer = false   # default = true (internal load balancer). You can set to false = public load balancer.
```

Comment:
Not yet implemented for the gcp.

Snuck it into https://github.com/MaterializeInc/materialize-terraform-self-managed/issues/129 since ...